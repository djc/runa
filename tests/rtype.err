rtype.rns [2.12]: return value type does not match declared return type
    '$str' vs 'int'
    return 's'
           ^
