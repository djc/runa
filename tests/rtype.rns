def do() -> int:
	return 's'
