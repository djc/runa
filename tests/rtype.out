tests/rtype.lng [2.9]: do() return type must be of type 'int'
    return 's'
           ^
